back/test/integration/TestTechMapsApi.js

```diff
@@ -63,26 +62,331 @@ describe("API", () => {
+
+      const getNewRes = await chai
+        .request(app.server())
+        .get(`/techMaps/${modifiedMap.id}`);
+
+      expect(getNewRes).to.have.status(200);
+      expect(getNewRes.body).to.deep.include(newTechMap);
+      expect(getNewRes.body).to.have.property("version");
+
+      const insertVersion = getNewRes.body.version;
+
+      const modifiedMap = {
+        ...newTechMap,
+        name: "Хліб Китайський (КХ)"
+      };
+
+      const putRes = await chai
+        .request(app.server())
+        .put(`/techMaps/${modifiedMap.id}`)
+        .type("application/json")
+        .send(JSON.stringify(modifiedMap));
+
+      expect(putRes).to.have.status(201);
+
+      const getRes = await chai.request(app.server()).get(`/techMaps/${modifiedMap.id}`);
+
+      expect(getRes).to.have.status(200);
+      expect(getRes.body).to.deep.include(modifiedMap);
+      expect(getNewRes.body).to.have.property("version");
+      expect(getNewRes.body.version).to.be.above(insertVersion);
     });
 
-    it.skip("should not bump up version when unmodified techmap put", async() => {
+    it("should not bump up version when unmodified techmap put", async () => {
       // todo: consider returning error or some not-modified for this case,
       // or emit warning at least, since it looks like clients logic error.
+      const id = newTechMapId();
+
+      const newTechMap = {
+        id,
+        name: "Хліб Французький (КХ)",
+        units: [1, 6],
+        steps: [
+          {
+            id: "STP-XXX-YYY-ZZZ-YY0",
+            name: "Замішування",
+            ingredients: [
+              {
+                ingredientId: "ING-XXX-YYY-ZZZ-YY0",
+                countByUnits: new Map([[1, 292], [6, 1752]])
+              }
+            ],
+            humanResources: [
+              {
+                peopleCount: 1,
+                countByUnits: new Map([[1, 15], [6, 22]])
+              }
+            ],
+            inventory: [
+              {
+                deviceId: "INV-XXX-YYY-ZZZ,YY0",
+                countByUnits: new Map([[1, 1], [6, 1]])
+              }
+            ],
+            instructions: `{"blocks":[{"key":"2ic2d","text":"Відважити воду та пшеничну закваску в чисту і суху ємність відповідного об'єму.","type":"ordered-list-item","depth":0,"inlineStyleRanges":[],"entityRanges":[],"data":{}},{"key":"7pdcs","text":"Окремо відважити боршно в чисту і суху ємність відповідного об’єму.","type":"ordered-list-item","depth":0,"inlineStyleRanges":[],"entityRanges":[],"data":{}},{"key":"an4s6","text":"В спіральному тістомісі змішати компоненти з послідовності (1) на протязі 4—5 хвилин (в залежності від кількості замісу).","type":"ordered-list-item","depth":0,"inlineStyleRanges":[],"entityRanges":[],"data":{}}],"entityMap":{}}`
+          }
+        ]
+      };
+
+      const insertRes = await chai
+        .request(app.server())
+        .post(`/techMaps/`)
+        .type("application/json")
+        .send(JSON.stringify(newTechMap));
```

Comment:
We should try not to use post in each test. This test is about modification, so post can be avoided and TM inserted directly via App. Motivation: if we brake just POST /techmaps/ we will have a lot of failed tests but ideally we should see failed only those tests that test POST /techmap/